examples/udp_client/linux_udp_client.cc

```diff
@@ -0,0 +1,135 @@
+/**************************************************************************/ /**
+ * @brief Simple Linux UDP client example.
+ * @file
+ *
+ * @note
+ * This is a minimal UDP client implementation, meant as an example of how to
+ * connect to a device and decode incoming data. It is not robust to network
+ * outages, socket reconnects, or other typical network errors. Production
+ * implementations must check for and handle these cases appropriately (by
+ * checking expected `errno` values, etc.).
+ ******************************************************************************/
+
+#include <cerrno>
+#include <cmath> // For lround()
+#include <csignal> // For signal()
+#include <cstdio> // For fprintf()
+#include <cstring> // For memcpy()
+#include <string> // For stoi() and strerror()
+
+#include <netdb.h> // For gethostbyname() and hostent
+#include <netinet/in.h> // For IPPROTO_* macros and htons()
+#include <sys/socket.h> // For socket support.
+#include <unistd.h> // For close()
+
+#include <iostream>
+#include <fstream>
```

Comment:
Sort along with the other includes above please. Standard includes (iostream, fstream) in the first group, system includes (inet.h) in teh second.

Reply:
done.